BREEZE.rst

```diff
@@ -263,8 +310,8 @@ to enter the running container. It's as easy as launching ``breeze exec`` while
 Breeze environment. You will be dropped into bash and environment variables will be read in the same
 way as when you enter the environment. You can do it multiple times and open as many terminals as you need.
 
-Stopping Breeze
----------------
+Stopping Interactive environment
```

Comment:
There is a reference to this section where you need to change it, too. see https://github.com/apache/airflow/blob/master/BREEZE.rst#stopping-the-environment

Reply:
Thks!

Reply:
I removed the reference. It was completely unnecessary :)